src/Nethermind/Nethermind.JsonRpc/Modules/Eth/FeeHistory/FeeHistoryOracle.cs

```diff
@@ -66,7 +75,7 @@ public ResultWrapper<FeeHistoryResults> GetFeeHistory(long blockCount, BlockPara
                 block = _blockFinder.FindParent(block, BlockTreeLookupOptions.RequireCanonical);
             }
 
-            FeeHistoryResults feeHistoryResults = new(oldestBlockNumber, baseFeePerGas.ToArray(), gasUsedRatio.ToArray(), rewards?.ToArray());
+            FeeHistoryResults feeHistoryResults = new(oldestBlockNumber, baseFeePerGas.ToArray(), gasUsedRatio.ToArray(), baseFeePerBlobGas.ToArray(), blobGasUsedRatio.ToArray(), rewards?.ToArray());
```

Comment:
Can we make FeeHistoryResult disposable, and change all collections to ArrayPoolList and dispose it there? We wouldn't have to allocate almost anything.

Reply:
lets do it in next PR